src/librustdoc/html/static/FiraSans-LICENSE.txt

```diff
@@ -24,7 +19,7 @@ with others.
 
 The OFL allows the licensed fonts to be used, studied, modified and
 redistributed freely as long as they are not sold by themselves. The
-fonts, including any derivative works, can be bundled, embedded,
+fonts, including any derivative works, can be bundled, embedded, 
```

Comment:
is the "random" trailing space to keep it inline with the original for some reason?